reporter.js

```diff
@@ -113,6 +113,12 @@ export function reportTest( test, { fullBrowser, id } ) {
 	}
 }
 
+export function reportError( error ) {
+	console.error( chalk.red( `\n\nError: ${ error.message }` ) );
```

Comment:
Nit I just noticed: We could show the real error type here, just like the browser does when thrown:
```suggestion
	console.error( chalk.red( `\n\${ error.name || "Error" }: ${ error.message }` ) );
```
I'm not sure if the fallback to `"Error"` is really needed but it doesn't hurt.

Reply:
I think it helps to see `ReferenceError` or `SyntaxError` instead of just `Error` here.

Reply:
I did essentially this, but made some other changes.